.circleci/config.yml

```diff
@@ -42,9 +42,9 @@ jobs:
           command: |
             make check-go-mod
 
-  gocode-check-via-gometalinter-swagger:
+  gocode-check-via-golangci-lint-swagger:
```

Comment:
In fact, there is no need to start a new job for `golangci-lint`. If so, the CI will pull the image `pouchcontainer/pouchlinter:v0.2.3` twice. I think add it to the job `markdownlint-misspell-shellcheck` and as a step is OK. WDYT?

Reply:
Agree